include/vcpkg/commands.build.h

```diff
@@ -42,7 +42,7 @@ namespace vcpkg
 
     struct CiBuildLogsRecorder final : IBuildLogsRecorder
     {
-        explicit CiBuildLogsRecorder(const Path& base_path_);
+        explicit CiBuildLogsRecorder(const Path& base_path_, int64_t minimum_last_write_time = 0);
```

Comment:
Does this need the = 0? It seems like all callers should be required to pass it.